packages/example/ios/example_0_70_6Tests/DataModelsTests/Banks Component/RNTBanksCollectableDataTests.swift

```diff
@@ -0,0 +1,105 @@
+//
+//  RNTBanksCollectableDataTests.swift
+//  example_0_70_6Tests
+//
+//  Created by Boris on 23.5.24..
+//
+
+import XCTest
+@testable import primer_io_react_native
+import PrimerSDK
+
+class BanksCollectableDataRNTests: XCTestCase {
+
+    func testFilterRNInitialization() {
+        let filter = FilterRN(text: "filterText")
+        XCTAssertEqual(filter.validatableDataName, "bankListFilter")
+        XCTAssertEqual(filter.text, "filterText")
+    }
+
+    func testBankIdRNInitialization() {
+        let bankId = BankIdRN(id: "bankId123")
+        XCTAssertEqual(bankId.validatableDataName, "bankId")
+        XCTAssertEqual(bankId.id, "bankId123")
+    }
+
+    func testBanksCollectableDataToBankIdRN() {
```

Comment:
Nit: This is a real nitpick, so feel free to ignore as functionally doesnt make a difference.

`if case let` could simplify these code blocks

```swift
let bankData = BanksCollectableData.bankId(bankId: "bankId123")
    
if case let .bankId(bankId) = bankData {
    let bankIdRN = BankIdRN(id: bankId)
    XCTAssertEqual(bankIdRN.validatableDataName, "bankId")
    XCTAssertEqual(bankIdRN.id, "bankId123")
} else {
    XCTFail("Conversion to BankIdRN failed")
}
```

Reply:
resolved. thanks! good point